### ISSUE TITLE
 accuracy values for the training 

### ISSUE THREAD

**Issue by hanmei1213** [Vincent-ZHQ/CA-MSER#11]
Hello, I printed the validation set accuracy and it is showing normally at around 71. However, the two accuracy values for the training set are both around 25, and I'm not sure why that is happening.

**Comment by wangdtfyguhjknjbhvgc** [comment]
Hi, I'm also experiencing this problem, has it been solved yet? And the accuracy curve of the training set is not quite normal, I even doubt the validation set accuracy anymore.

**Comment by wangdtfyguhjknjbhvgc** [comment]
Hello, I encountered this issue as well. Has it been resolved? Also, the accuracy curve of the training set doesn't look quite right, and I even suspect the accuracy of the validation set.

**Comment by wangdtfyguhjknjbhvgc** [comment]
Hello, I encountered this issue as well. Has it been resolved? Also, the accuracy curve of the training set doesn't look quite right, and I even suspect the accuracy of the validation set.
